src/js/features/progress.js

```diff
@@ -312,7 +346,9 @@ Object.assign(MediaElementPlayer.prototype, {
 		media.addEventListener('timeupdate', (e) => {
 			if (media.duration !== Infinity ) {
 				player.setProgressRail(e);
-				player.setCurrentRail(e);
+				if(!t.forcedHandlePause){
```

Comment:
so if you add this option `t.option.lowMemoryOptimize` then condition should  change to 

if(!t.forcedHandlePause || t.option.lowMemoryOptimize){
 ...
}

Same for progress event

.

See below for t.option.lowMemoryOptimize